ios/rntemplate.xcodeproj/xcshareddata/xcschemes/rntemplate-tvOS.xcscheme

```diff
@@ -55,6 +55,15 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
What do you think about this tvOS scheme? I think we are safe to remove it, any of our projects are targeted for tvOS 🤷

Reply:
I will remove it 👍